db/migrate/20211027140313_migrate_customer_names.rb

```diff
@@ -0,0 +1,28 @@
+class MigrateCustomerNames < ActiveRecord::Migration[6.1]
+  class ApplicationRecord < ActiveRecord::Base
+    self.abstract_class = true
+  end
+  class Enterprise < ApplicationRecord; end
```

Comment:
Just an idea:

```suggestion
  class Enterprise < ApplicationRecord
    scope showing_customer_names: -> do
      joins("JOIN spree_preferences ON value LIKE '--- true\n%' AND spree_preferences.key = CONCAT('/enterprise/show_customer_names_to_suppliers/', enterprises.id)")
    end
  end
```

It would enable us to update all those enterprises at once.